week-5/Homework/mandatory/3-project/js/main.js

```diff
@@ -0,0 +1,93 @@
+let blueBtn =  document.getElementById("blueBtn");
+let orangeBtn =  document.getElementById("orangeBtn");
+let greenBtn =  document.getElementById("greenBtn");
+let submitBtn = document.querySelector("form button");
+let jumbotron = document.querySelector(".jumbotron");
+let donate = document.getElementsByClassName("btn btn-primary btn-lrg");
+let volunteer = document.getElementsByClassName("btn btn-secondary btn-lrg");
+let emailInput = document.getElementById("exampleInputEmail1");
+let nameInput = document.getElementById("example-text-input");
+let descrInput = document.getElementById("exampleTextarea");
+let form =  document.querySelector("form");
+
+
+
+let blueBtnClick = function (){
+
+    jumbotron.style.backgroundColor = "#588fbd";
+    donate[0].style.backgroundColor = "#ffa500";
+    volunteer[0].style.backgroundColor = "black";
+    volunteer[0].style.color = "white";
+}
+
+blueBtn.addEventListener("click", blueBtnClick);
+
+
+let orangeBtnClick = function (){
+
+    jumbotron.style.backgroundColor = "#f0ad4e";
+    donate[0].style.backgroundColor = "#5751fd";
+    volunteer[0].style.backgroundColor = "#31b0d5";
+    volunteer[0].style.color = "white";
+}
+
+orangeBtn.addEventListener("click", orangeBtnClick);
+
+let greenBtnClick = function (){
+
+    jumbotron.style.backgroundColor = "#87ca8a";
+    donate[0].style.backgroundColor = "black";
+    volunteer[0].style.backgroundColor = "#8c9c08";
+    volunteer[0].style.color = "white";
+}
+greenBtn.addEventListener("click", greenBtnClick);
+
+
+let mailCheck = function(){
+   
+    if ((emailInput.value.length === 0 ) || ( !emailInput.value.includes("@") ))
```

Comment:
How about keeping this to only do what it infers by its name. Only check if the mail is valid and move the styling of the `emailInput` elsewhere.

Also, the function can become a lot shorter considering all you need to return is
`return (emailInput.value.length > 0 ) && emailInput.value.includes("@")`
your check function can be as simple as
```
let isEmailValid = () => (emailInput.value.length > 0 ) && emailInput.value.includes("@")
```
considering you may need this same function elsewhere, we could be passing the input as an argument to your function. 
```
let isEmailValid = (emailValue) => (emailValue.length > 0 ) && emailValue.includes("@")
```